code/modules/uplink/uplink_items/uplink_roles.dm

```diff
@@ -28,16 +28,16 @@
 			blast wave \"projectile\". Aspiring scientists may find this highly useful, as forcing the pressure shockwave into a narrow angle seems to be able to bypass whatever quirk of physics \
 			disallows explosive ranges above a certain distance, allowing for the device to use the theoretical yield of a transfer valve bomb, instead of the factual yield."
 	item = /obj/item/gun/blastcannon
-	cost = 14							//High cost because of the potential for extreme damage in the hands of a skilled gas masked scientist.
+	cost = 13							//High cost because of the potential for extreme damage in the hands of a skilled gas masked scientist.
 	restricted_roles = list("Research Director", "Scientist")
 
-/datum/uplink_item/role_restricted/alientech
-	name = "Alien Research Disk"
-	desc = "A technology disk holding a terabyte of highly confidential abductor technology. \
-			Simply insert into research console of choice and import the files from the disk. Because of its foreign nature, it may require multiple uploads to work properly."
-	item = /obj/item/disk/tech_disk/abductor
-	cost = 12
-	restricted_roles = list("Research Director", "Scientist", "Roboticist")
+/datum/uplink_item/role_restricted/brainwash_disk
+	name = "Brainwashing Surgery Program"
+	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
+	Insert into an Operating Console to enable the procedure."
+	item = /obj/item/disk/surgery/brainwashing
+	restricted_roles = list("Medical Doctor")
+	cost = 3
```

Comment:
Don't undo previous PRs without a reason

Reply:
A: Please stop backseat maintaining since you are not actually being very helpful with your feedback.
B: This was a port and accidental replacements are destined to happen, please tell me WHAT PR this undid so I can properly fix it.

Reply:
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10077